### PR TITLE
#711: removed any type from projects-get-all

### LINKED ISSUE #711: [Refactoring] - Remove `any` type from projects-get-all
### Description

Remove the use of the `any` type from projects-get-all. This may seem very simple, but the point of this is just to have you read through the codebase and understand the types/interfaces used in it, or make new ones that fit the shape of the data being used.

### Notes

Check #665 for notes on why we're doing this and pointers on what you should replace `any` with.

### Acceptance Criteria

`any` is not used as a type in projects-get-all.

### Proposed Solution

`Ctrl + F`

### PR DESCRIPTION
**Pull request by sewardwi**
## Changes

removed the any type on line 20 and replaced it with a WP array

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #711

### CODE REVIEW

File: src/shared/src/backend-supports/projects-get-all.ts [hunk 1]
```diff
@@ -12,7 +12,12 @@ const calculateEndDate = (start: Date, weeks: number) => {
   return end;
 };
 
-const projectDurationBuilder = (wps: any) => {
+interface WP {
+  startDate: any;
```
Comment (callmejim1226) [inline src/shared/src/backend-supports/projects-get-all.ts:16]: The goal for this ticket is to remove **any** type in projects-get-all.ts. Can you replace the data type of startDate with the correct type? (Srry if this confused you last night, I just wrote **any** for this field on your laptop to quickly show you an property example we need for the work package)

File: src/shared/src/backend-supports/projects-get-all.ts [hunk 2]
```diff
@@ -12,7 +12,12 @@ const calculateEndDate = (start: Date, weeks: number) => {
   return end;
 };
 
-const projectDurationBuilder = (wps: any) => {
+interface WP {
```
Comment (callmejim1226) [inline src/shared/src/backend-supports/projects-get-all.ts:15]: Can we rename this interface as **WorkPackageDurationProps** to give more details for the code readers? I'm unsure if Reid or Anthony will have a different opinion on naming this type.

Comment (callmejim1226) [inline src/shared/src/backend-supports/projects-get-all.ts:15]: Also, can we locate this interface before the calculateEndDate function? For our file formatting, we wanna place the interface in the beginning after the imports.

**Review by callmejim1226 — CHANGES_REQUESTED** [review]
Great start!!

I commented few changes to improve our code for this ticket. Lmk if you need anything for this!

**Review by callmejim1226 — APPROVED** [review]
The code looks good to me! Great work! (Idk why prettier test is failing, but make sure your prettier extension is enabled on your VScode)

**Review by anthonybernardi — APPROVED** [review]
nice, dont forget the "#" in teh branch name

**Pull request merged**